### ISSUE TITLE
Discs: Kodi can not eject from the right drive

### ISSUE THREAD

**Issue by CS-Alchemist** [xbmc/xbmc#22059]
### Describe the bug
Here is a clear and concise description of what the problem is:

As described in the headline, Kodi is not able to eject a disc from the correct drive. This of course occurs when a system has two or more drives connected.
In my case I have two SATA drives under Linux.
/dev/sr0 is a DVD drive and /dev/sr1 is a BR drive.
When I put a disc in my BR drive, and want to eject that disc under Kodi, it always ejects the DVD drive.

just for fun i had a closer look at the function ```void CDiscDriveHandlerPosix::EjectDriveTray(const std::string& devicePath)```. The passing parameter in my case is always "/dev/cdrom". But /dev/cdrom is not a real block device, it is a symlink to /dev/sr0. Should Kodi use symlinks at all? Why doesn't Kodi recognize the correct block device /dev/sr1?

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

- Kodi should eject the Disc from the correct drive

## Actual Behavior

- Kodi allways eject /dev/cdrom -> /dev/sr0

## Possible Fix

There is no fix in Kodi

### To Reproduce

1. Use a PC with min. 2 Drives
2. Insert a Disc in bouth drives
3. Try to eject those disc with kodi

### Debuglog

The debuglog can be found here:
[kodi.log](https://github.com/xbmc/xbmc/files/9844718/kodi.log)

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [x] Linux
 - [ ] OSX
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Debian Bullseye
 - Kodi version:
 Kodi 20.0-ALPHA3(19.90.710)
 GIT: 58e6ecb639b350a503d2e467602e6ecde4423342

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by enen92** [comment]
Yes this is not supported, ATM Kodi only supports a single drive
